setup.py

```diff
@@ -12,7 +12,10 @@
     author="Algorand",
     author_email="pypiservice@algorand.com",
     python_requires=">=3.9",
-    install_requires=["py-algorand-sdk>=2.0.0", "tabulate==0.9.0"],
+    install_requires=[
+        "py-algorand-sdk@git+https://github.com/algorand/py-algorand-sdk@better-exports",
+        "tabulate==0.9.0",
+    ],
```

Comment:
^^^ TODO: revert to correct version before merging ^^^

Reply:
@ahangsu - I'm giving up on this for now. But I created #52 so it gets done ASAP.

Reply:
👍 I am fine either way, and releasing sdk is beyond us. We can merge as is and pin the right version after sdk release, is that fine with you?

Reply:
Agreed and resolving this confo